### PR TITLE
[Reduce dependence]>[Lists] Replace "google.common.collect.*" API wit…

### PR DESCRIPTION
**Pull request by zzq1314zll**
Replace "google.common.collect.Lists" 
for #6402

### CODE REVIEW

File: common/src/main/java/com/alibaba/nacos/common/utils/CollectionUtils.java [hunk 1]
```diff
@@ -245,4 +247,22 @@ public static boolean isNotEmpty(Collection coll) {
...
+     * Return an arraylist containing all input parameters.
+     * Throw an NullPointerException when the input parameter is null.
+     * @author zzq
+     * @param elements element array
+     * @return arraylist containing all input parameters
+     *
+     */
+    public static <T> List<T> list(T... elements) {
+        if (elements == null) {
+            throw new NullPointerException();
```
Comment (KomachiSion) [inline common/src/main/java/com/alibaba/nacos/common/utils/CollectionUtils.java:261]: Whether `IllegalArgumentException` is better?

File: common/src/main/java/com/alibaba/nacos/common/utils/CollectionUtils.java [hunk 2]
```diff
@@ -245,4 +247,22 @@ public static boolean isNotEmpty(Collection coll) {
...
+        }
+        ArrayList<T> list = new ArrayList<>();
```
Comment (KomachiSion) [inline common/src/main/java/com/alibaba/nacos/common/utils/CollectionUtils.java:263]: use elements.size to init the array size?

**Review by KomachiSion — CHANGES_REQUESTED** [review]

**Review by KomachiSion — DISMISSED** [review]

**Comment by KomachiSion** [comment]
There are some conflict, please solve them.

**Comment by zzq1314zll** [comment]
@KomachiSion Already solved

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**